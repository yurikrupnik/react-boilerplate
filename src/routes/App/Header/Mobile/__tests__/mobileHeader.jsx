import React from 'react';
import Component from '../index';
import {  } from 'react-router-dom';
import { StaticRouter } from "react-router-dom";

const {
    it,
    expect,
    mount
} = global;

it('renders <Component /> component', () => {
    const wrapper = mount((
        <StaticRouter context={{}}>
            <Component />
        </StaticRouter>
    ));
    expect(wrapper).toMatchSnapshot();
});
