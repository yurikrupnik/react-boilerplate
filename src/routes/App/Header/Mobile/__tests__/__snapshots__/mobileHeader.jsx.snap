// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Component /> component 1`] = `
<Mobile>
  <SidebarConsumer
    render={[Function]}
  >
    <WithStyles(AppBar)
      position="static"
    >
      <AppBar
        classes={
          Object {
            "colorDefault": "MuiAppBar-colorDefault-7",
            "colorPrimary": "MuiAppBar-colorPrimary-8",
            "colorSecondary": "MuiAppBar-colorSecondary-9",
            "positionAbsolute": "MuiAppBar-positionAbsolute-3",
            "positionFixed": "MuiAppBar-positionFixed-2",
            "positionRelative": "MuiAppBar-positionRelative-6",
            "positionStatic": "MuiAppBar-positionStatic-5",
            "positionSticky": "MuiAppBar-positionSticky-4",
            "root": "MuiAppBar-root-1",
          }
        }
        color="primary"
        position="static"
      >
        <WithStyles(Paper)
          className="MuiAppBar-root-1 MuiAppBar-positionStatic-5 MuiAppBar-colorPrimary-8"
          component="header"
          elevation={4}
          square={true}
        >
          <Paper
            className="MuiAppBar-root-1 MuiAppBar-positionStatic-5 MuiAppBar-colorPrimary-8"
            classes={
              Object {
                "elevation0": "MuiPaper-elevation0-12",
                "elevation1": "MuiPaper-elevation1-13",
                "elevation10": "MuiPaper-elevation10-22",
                "elevation11": "MuiPaper-elevation11-23",
                "elevation12": "MuiPaper-elevation12-24",
                "elevation13": "MuiPaper-elevation13-25",
                "elevation14": "MuiPaper-elevation14-26",
                "elevation15": "MuiPaper-elevation15-27",
                "elevation16": "MuiPaper-elevation16-28",
                "elevation17": "MuiPaper-elevation17-29",
                "elevation18": "MuiPaper-elevation18-30",
                "elevation19": "MuiPaper-elevation19-31",
                "elevation2": "MuiPaper-elevation2-14",
                "elevation20": "MuiPaper-elevation20-32",
                "elevation21": "MuiPaper-elevation21-33",
                "elevation22": "MuiPaper-elevation22-34",
                "elevation23": "MuiPaper-elevation23-35",
                "elevation24": "MuiPaper-elevation24-36",
                "elevation3": "MuiPaper-elevation3-15",
                "elevation4": "MuiPaper-elevation4-16",
                "elevation5": "MuiPaper-elevation5-17",
                "elevation6": "MuiPaper-elevation6-18",
                "elevation7": "MuiPaper-elevation7-19",
                "elevation8": "MuiPaper-elevation8-20",
                "elevation9": "MuiPaper-elevation9-21",
                "root": "MuiPaper-root-10",
                "rounded": "MuiPaper-rounded-11",
              }
            }
            component="header"
            elevation={4}
            square={true}
          >
            <header
              className="MuiPaper-root-10 MuiPaper-elevation4-16 MuiAppBar-root-1 MuiAppBar-positionStatic-5 MuiAppBar-colorPrimary-8"
            >
              <div>
                <WithStyles(Button)>
                  <Button
                    classes={
                      Object {
                        "colorInherit": "MuiButton-colorInherit-58",
                        "contained": "MuiButton-contained-48",
                        "containedPrimary": "MuiButton-containedPrimary-49",
                        "containedSecondary": "MuiButton-containedSecondary-50",
                        "disabled": "MuiButton-disabled-57",
                        "extendedFab": "MuiButton-extendedFab-55",
                        "fab": "MuiButton-fab-54",
                        "flat": "MuiButton-flat-42",
                        "flatPrimary": "MuiButton-flatPrimary-43",
                        "flatSecondary": "MuiButton-flatSecondary-44",
                        "focusVisible": "MuiButton-focusVisible-56",
                        "fullWidth": "MuiButton-fullWidth-62",
                        "label": "MuiButton-label-38",
                        "mini": "MuiButton-mini-59",
                        "outlined": "MuiButton-outlined-45",
                        "outlinedPrimary": "MuiButton-outlinedPrimary-46",
                        "outlinedSecondary": "MuiButton-outlinedSecondary-47",
                        "raised": "MuiButton-raised-51",
                        "raisedPrimary": "MuiButton-raisedPrimary-52",
                        "raisedSecondary": "MuiButton-raisedSecondary-53",
                        "root": "MuiButton-root-37",
                        "sizeLarge": "MuiButton-sizeLarge-61",
                        "sizeSmall": "MuiButton-sizeSmall-60",
                        "text": "MuiButton-text-39",
                        "textPrimary": "MuiButton-textPrimary-40",
                        "textSecondary": "MuiButton-textSecondary-41",
                      }
                    }
                    color="default"
                    component="button"
                    disableFocusRipple={false}
                    disabled={false}
                    fullWidth={false}
                    mini={false}
                    size="medium"
                    type="button"
                    variant="text"
                  >
                    <WithStyles(ButtonBase)
                      className="MuiButton-root-37 MuiButton-text-39 MuiButton-flat-42"
                      component="button"
                      disabled={false}
                      focusRipple={true}
                      focusVisibleClassName="MuiButton-focusVisible-56"
                      type="button"
                    >
                      <ButtonBase
                        centerRipple={false}
                        className="MuiButton-root-37 MuiButton-text-39 MuiButton-flat-42"
                        classes={
                          Object {
                            "disabled": "MuiButtonBase-disabled-64",
                            "focusVisible": "MuiButtonBase-focusVisible-65",
                            "root": "MuiButtonBase-root-63",
                          }
                        }
                        component="button"
                        disableRipple={false}
                        disableTouchRipple={false}
                        disabled={false}
                        focusRipple={true}
                        focusVisibleClassName="MuiButton-focusVisible-56"
                        tabIndex="0"
                        type="button"
                      >
                        <button
                          className="MuiButtonBase-root-63 MuiButton-root-37 MuiButton-text-39 MuiButton-flat-42"
                          disabled={false}
                          onBlur={[Function]}
                          onFocus={[Function]}
                          onKeyDown={[Function]}
                          onKeyUp={[Function]}
                          onMouseDown={[Function]}
                          onMouseLeave={[Function]}
                          onMouseUp={[Function]}
                          onTouchEnd={[Function]}
                          onTouchMove={[Function]}
                          onTouchStart={[Function]}
                          tabIndex="0"
                          type="button"
                        >
                          <span
                            className="MuiButton-label-38"
                          >
                            toggle
                          </span>
                          <NoSsr
                            defer={false}
                            fallback={null}
                          >
                            <WithStyles(TouchRipple)
                              center={false}
                              innerRef={[Function]}
                            >
                              <TouchRipple
                                center={false}
                                classes={
                                  Object {
                                    "child": "MuiTouchRipple-child-70",
                                    "childLeaving": "MuiTouchRipple-childLeaving-71",
                                    "childPulsate": "MuiTouchRipple-childPulsate-72",
                                    "ripple": "MuiTouchRipple-ripple-67",
                                    "ripplePulsate": "MuiTouchRipple-ripplePulsate-69",
                                    "rippleVisible": "MuiTouchRipple-rippleVisible-68",
                                    "root": "MuiTouchRipple-root-66",
                                  }
                                }
                              >
                                <TransitionGroup
                                  childFactory={[Function]}
                                  className="MuiTouchRipple-root-66"
                                  component="span"
                                  enter={true}
                                  exit={true}
                                >
                                  <span
                                    className="MuiTouchRipple-root-66"
                                  />
                                </TransitionGroup>
                              </TouchRipple>
                            </WithStyles(TouchRipple)>
                          </NoSsr>
                        </button>
                      </ButtonBase>
                    </WithStyles(ButtonBase)>
                  </Button>
                </WithStyles(Button)>
              </div>
            </header>
          </Paper>
        </WithStyles(Paper)>
      </AppBar>
    </WithStyles(AppBar)>
  </SidebarConsumer>
</Mobile>
`;
